ramls/request.json

```diff
@@ -226,6 +226,11 @@
       "type": "string",
       "format": "date-time",
       "readonly" : true
+    },
+    "searchIndex": {
+      "description": "Request fields used for search",
+      "type": "object",
+      "$ref": "searchIndex.json"
```

Comment:
This filename looks too generic. What if we add searchIndex to other objects that would have different indexing fields? We can rename it to requestSearchIndex.json

Reply:
thanks, renamed it